compose/bin/docker-compose

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if hash docker-compose 2>/dev/null; then
+  DOCKER_COMPOSE="docker-compose"
+else
+  DOCKER_COMPOSE="docker compose"
+fi
+
+if [ "$1" == "--no-dev-up" ]; then
+  ${DOCKER_COMPOSE} -f docker-composer.yml up -d
```

Comment:
Should we also provide the ability to pass commands with `--no-dev-up` by adding `"$@"` at the end?

Reply:
Use `--no-dev` and `"${@:2}"`